### PR TITLE
Add a custom object in HyBridge initialization

### PR DESCRIPTION
**Pull request by joserg**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,17 +20,20 @@ Pods
...
 .DS_Store
-/.settings
+.settings
 
 *.orig
 .picasa.ini
 
 # Android
 */*/.classpath
+*/*/*/.classpath
```
Comment (davidgarsan) [inline .gitignore]: Sure?

File: android/Hybridge/src/com/pdi/hybridge/HybridgeBroadcaster.java [hunk 2]
```diff
@@ -32,11 +32,12 @@ private HybridgeBroadcaster() {
         mJsBuffer = new StringBuffer("");
     }
 
-    public void initJs(WebView view, JSONArray actions, JSONArray events) {
+    public void initJs(WebView view, JSONArray actions, JSONArray events, JSONObject custom) {
```
Comment (davidgarsan) [inline android/Hybridge/src/com/pdi/hybridge/HybridgeBroadcaster.java]: Everything is custom (actions, events...) Maybe would be better a more descriptive name. The same all along the code (JS, Android, iOS).

**Comment by davidgarsan** [comment]
LGTM

**Comment by shinyieva** [comment]
LGTM

**Comment by jmendiara** [comment]
LGTM

**Comment by dcantelar** [comment]
LGTM

**Pull request merged**